README.md

```diff
@@ -102,7 +135,30 @@ WNFS exposes a familiar POSIX-style interface:
 - `mv`: move a file or directory
 - `read`: alias for `cat`
 - `rm`: remove a file or directory
-- `write`: write to a file
+- `write`: alias for `add`
+
+
+## Publicise
+
+The `publicise` function synchronises your file system with the Fission API and IPFS. We don't do this automatically because if you add a large set of data, you only want to do this after everything is added. Otherwise it would be too slow and we would have too many network requests to the API.
```

Comment:
think I used american spelling here :thinking: feel free to switch it/alias on rebase

Reply:
ah you already did that. nvm!